web-ng/angular.json

```diff
@@ -24,6 +24,7 @@
               "src/assets"
             ],
             "styles": [
+              "./node_modules/@angular/material/prebuilt-themes/indigo-pink.css",
```

Comment:
Please remove the indigo-pink.css here and below - we'll use our own custom style in src/styles.css.